form.php

```diff
@@ -0,0 +1,55 @@
+<?php
+    $fullName = "";
+    $flag = 0;
+    $trimmed = "";
+    $namePattern = '/^[a-zA-Z]+$/';
+    if ($_SERVER['REQUEST_METHOD'] == 'POST' && isset ($_POST['Submit'])) {
+        $fName = $_POST['fName'];
+        $lName = $_POST['lName'];
+        $fullName = $fName." ".$lName;
+        $trimmed = preg_replace ($namePattern, " ", $fullName);
+        if (preg_match ($namePattern,$fName) && preg_match ($namePattern,$lName)) {
```

Comment:
It should be written like this ->
`if(preg_match($namePattern, $fName) && preg_match($namePattern ,$lName)) {`